styles/style-main.css

```diff
@@ -154,6 +170,20 @@ h1 {
   padding-inline-start: 0;
 }
 
+.technologies li {
+  float: left;
+}
+
+.contact-me-form li {
+  list-style: none;
+  border: solid gray;
+  border-radius: 5px;
```

Comment:
- [ ] I think it would be a wonderful idea to make the border of all input fields `1px solid #dfdfe0` and the outlines can have a color of `#1a2236` as seen in the Figma template that you are using. Kindly check below for further clarifications:

## Your Design

![image](https://user-images.githubusercontent.com/77106962/144140303-358cbe04-557b-45fd-97ca-7c765e1d5bf0.png)

## Figma Design 

![image](https://user-images.githubusercontent.com/77106962/144140369-46593f09-ca4b-4e20-af55-4f5f042bc3e0.png)